content-services/content-sources-backend
# Fixes 2188: Add more detail in snapshot listing

## Summary
Adds more detail to Snapshots and Repository response: 

- Adds added_counts & removed_counts to the snapshots table/SnapshotResponse

- Adds last_snapshot_uuid and last_snapshot to the repository_configurations table/RepositoryResponse

## Testing steps
- Add a repository with snapshotting enabled. 
- Fetch/list/edit said repository after it has finished snapshotting, one should see the above new fields on the response.
- Call the snapshot list api, one should see the associated fields above on the response.
